### PR TITLE
Upgrade cAdvisor version in k8s deployment.

### PR DESCRIPTION
**Pull request by Creatone**
Signed-off-by: Paweł Szulik <pawel.szulik@intel.com>

### CODE REVIEW

File: deploy/kubernetes/base/daemonset.yaml [hunk 1]
```diff
@@ -17,14 +17,14 @@ spec:
       serviceAccountName: cadvisor
       containers:
       - name: cadvisor
-        image: k8s.gcr.io/cadvisor:v0.30.2
+        image: gcr.io/cadvisor/cadvisor:v0.39.0
         resources:
           requests:
-            memory: 200Mi
-            cpu: 150m
+            memory: 1000Mi
```
Comment (bobbypage) [inline deploy/kubernetes/base/daemonset.yaml:23]: just curious, why is the bump in cpu and memory needed?

Comment (Creatone) [reply deploy/kubernetes/base/daemonset.yaml:23]: Got this one:
```
[pszulik@node200 ~]$ curl 10.233.79.136:8080/metrics > out
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (7) Failed connect to 10.233.79.136:8080; Connection refused

```
Lack of resources...

Comment (bobbypage) [reply deploy/kubernetes/base/daemonset.yaml:23]: I see, sounds good to bump then. However, it seems like a large bump (requests 200Mi -> 1000Mi ; cpu 100m -> 500m). I don't have any objections but just curious how you picked those... or if it's just a rough guess :)

Comment (iwankgb) [reply deploy/kubernetes/base/daemonset.yaml:23]: It has not been changed for a while but such a huge increase is a bit worrying to me too. @Creatone have you tried to set any lower value?

Comment (dashpole) [reply deploy/kubernetes/base/daemonset.yaml:23]: Resources are going to vary based on the number of containers on the node.
You should also check to see if there were any additional --disable_metrics added during that time period that we need to ignore (if we are ignoring any).

Comment (Creatone) [reply deploy/kubernetes/base/daemonset.yaml]: Can you now check out the values?

**Comment by iwankgb** [comment]
@bobbypage there are few recently reported issues about performance. I think we can merge this PR and investigate reported issues.

**Review by iwankgb — APPROVED** [review]

**Comment by bobbypage** [comment]
Sounds good, LGTM.

**Pull request merged**